pkg/commands/entrypoints/entrypoints.go

```diff
@@ -88,6 +88,13 @@ func (c *EntrypointsCommand) Flags() *cli.FlagSet {
 
 	f := set.NewSection("COMMAND OPTIONS")
 
+	f.StringVar(&cli.StringVar{
+		Name:    "dir",
+		Target:  &c.flagDir,
+		Example: "./terraform",
```

Comment:
Should this default to CWD?

Reply:
+1, we should make this consistent throughout.